glyphs-reader/src/font.rs

```diff
@@ -67,15 +67,33 @@ pub struct Glyph {
     pub other_stuff: BTreeMap<String, Plist>,
 }
 
+/// Represents a path OR a component
```

Comment:
why not an enum then? As as struct, it actually represent both a path AND a component which is weird

Reply:
I guess if we made Shape an enum, we would have to implement a custom FromPlist trait to be able to parse Layer::shapes into this Vec<Shape>, whereas with a simple struct you can simply derive FromPlist?

Reply:
Yes. Perhaps that woudl be worth doing.

Reply:
Per IM discussion will do so

Reply:
Introduced an enum, it's much nicer now. Good call. Also added use of TryFrom which I think makes the code flow a little better.